features/step_definitions/protected_area_steps.rb

```diff
@@ -0,0 +1,3 @@
+Given(/^there is a user with username "([^"]*)" and password "([^"]*)"$/) do |usern, passw|
+  User.create(username: usern, password: passw)
```

Comment:
I would prefer these variables be `username` and `password`. In Ruby we are always striving to make readable code.